bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -273,6 +344,17 @@ pub mod pallet {
 			Ok(call)
 		}
 
+		// Build the call to dispatch the `EthereumSystem::add_tip` extrinsic on BH
+		fn build_add_tip_call(
+			message_id: MessageId,
+			amount: u128,
+		) -> BridgeHubRuntime {
+			BridgeHubRuntime::EthereumSystem(EthereumSystemCall::AddTip {
```

Comment:
Should also forward the original sender `who` to system-v2. It could be useful to have that information there.